mmv1/third_party/terraform/tests/resource_tags_test.go

```diff
@@ -879,3 +880,76 @@ func testAccCheckTagsLocationTagBindingDestroyProducer(t *testing.T) func(s *ter
 		return nil
 	}
 }
+
+
+
+func testAccTagsLocationTagBinding_locationTagBindingzonal(t *testing.T) {
```

Comment:
Thanks for adding the tests, can you make the first letter capital case? 
`TestAccTagsLocationTagBinding_locationTagBindingzonal`